commit 0e555968ea51ab125a070df10ed24ce0eb24ed2a
Author: davidsansome <davidsansome@94c5599e-fc6c-11de-b061-8119ef04aefe>
Date:   Mon May 30 14:54:11 2011 +0000

    Hack the wrapped ~QStandardItem to remove the row properly from the model (which deletes it as well)
    
    git-svn-id: https://clementine-player.googlecode.com/svn/trunk@3345 94c5599e-fc6c-11de-b061-8119ef04aefe

diff --git a/3rdparty/pythonqt/generated_cpp/com_trolltech_qt_gui/com_trolltech_qt_gui7.cpp b/3rdparty/pythonqt/generated_cpp/com_trolltech_qt_gui/com_trolltech_qt_gui7.cpp
index e08ea9b..16c24dc 100644
--- a/3rdparty/pythonqt/generated_cpp/com_trolltech_qt_gui/com_trolltech_qt_gui7.cpp
+++ b/3rdparty/pythonqt/generated_cpp/com_trolltech_qt_gui/com_trolltech_qt_gui7.cpp
@@ -1039,6 +1039,14 @@ QStandardItem* PythonQtWrapper_QStandardItem::new_QStandardItem(int  rows, int
 { 
 return new PythonQtShell_QStandardItem(rows, columns); }
 
+void PythonQtWrapper_QStandardItem::delete_QStandardItem(QStandardItem* obj) {
+  if (obj && obj->parent()) {
+    obj->parent()->removeRow(obj->row());
+  } else if (obj) {
+    delete obj;
+  }
+}
+
 QString  PythonQtWrapper_QStandardItem::accessibleDescription(QStandardItem* theWrappedObject) const
 {
   return ( theWrappedObject->accessibleDescription());
diff --git a/3rdparty/pythonqt/generated_cpp/com_trolltech_qt_gui/com_trolltech_qt_gui7.h b/3rdparty/pythonqt/generated_cpp/com_trolltech_qt_gui/com_trolltech_qt_gui7.h
index 23ff35d..380b7fc 100644
--- a/3rdparty/pythonqt/generated_cpp/com_trolltech_qt_gui/com_trolltech_qt_gui7.h
+++ b/3rdparty/pythonqt/generated_cpp/com_trolltech_qt_gui/com_trolltech_qt_gui7.h
@@ -161,7 +161,7 @@ QStandardItem* new_QStandardItem();
 QStandardItem* new_QStandardItem(const QIcon&  icon, const QString&  text);
 QStandardItem* new_QStandardItem(const QString&  text);
 QStandardItem* new_QStandardItem(int  rows, int  columns = 1);
-void delete_QStandardItem(QStandardItem* obj) { delete obj; } 
+void delete_QStandardItem(QStandardItem* obj);
    QString  accessibleDescription(QStandardItem* theWrappedObject) const;
    QString  accessibleText(QStandardItem* theWrappedObject) const;
    void appendColumn(QStandardItem* theWrappedObject, const QList<QStandardItem* >&  items);
